.github/workflows/tests.yaml

```diff
@@ -0,0 +1,23 @@
+name: Tests
+
+on: [push, workflow_dispatch]
```

Comment:
I think this isn't quite right -- it's needs to be `workflow_call` I think?

Reply:
Oops! Thanks! done in https://github.com/hashicorp/vault-workflows-common/pull/5/commits/137b6bfb76e0506b045aa9b81aea1a3cc6483b44